templates/jitsi_meet_config.js.j2

```diff
@@ -180,7 +180,7 @@ var config = {
     desktopSharingChromeMinExtVersion: '0.1',
 
     // Whether desktop sharing should be disabled on Firefox.
-    desktopSharingFirefoxDisabled: {{ jitsi_meet_desktop_sharing_firefox_disabled }},
+    desktopSharingFirefoxDisabled: {{ not ( jitsi_meet_enable_desktop_sharing_firefox | bool ) | lower }},
```

Comment:
Here it should be `{{ (not jitsi_meet_enable_desktop_sharing_firefox) | bool | lower }}`, otherwise the result is `False` instead of `false` and config.json can't be parsed correctly